### PR TITLE
Legacy: Add back legacy ctrl so you can enable app

### PR DESCRIPTION
**Pull request by torkelo**
Currently there is no way to enable app (view config page) without a angular config controller

### CODE REVIEW

File: src/dashboards/stats.json [hunk 1]
```diff
@@ -85,8 +85,8 @@
       ],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Panel Title",
-      "type": "singlestat2"
+      "title": "Panel Title3",
```
Comment (marcusolsson) [inline src/dashboards/stats.json:88]:
```suggestion
      "title": "Panel Title",
```

File: src/dashboards/streaming.json [hunk 2]
```diff
@@ -58,8 +58,8 @@
       ],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Simple dummy streaming example",
-      "type": "graph2"
+      "title": "Simple dummy streaming example4",
```
Comment (marcusolsson) [inline src/dashboards/streaming.json:61]:
```suggestion
      "title": "Simple dummy streaming example",
```

**Review by marcusolsson — APPROVED** [review]
LGTM. Is there a reason for suffixing numbers in the panel titles?

**Comment by torkelo** [comment]
> LGTM. Is there a reason for suffixing numbers in the panel titles?

 no was testing the automatic app dashboards update process

File: src/legacy/config.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { PluginMeta } from '@grafana/data';
+
+export class ExampleConfigCtrl {
+  static template = '<h2><hello/h2>';
```
Comment (s-h-a-d-o-w) [inline src/legacy/config.ts:4]:
```suggestion
  static template = '<h2>hello</h2>';
```

**Pull request merged**